src/Symfony/Component/Notifier/Bridge/LightSms/LightSmsTransport.php

```diff
@@ -124,8 +124,8 @@ protected function doSend(MessageInterface $message): SentMessage
 
         $content = $response->toArray(false);
 
-        if (0 !== (int) $content[$phone]['error'] ?? -1) {
-            $errorCode = (int) $content['error'] ?? $content['']['error'] ?? $content[$phone]['error'] ?? -1;
+        if (0 !== (int) ($content['error'] ?? $content['']['error'] ?? $content[$phone]['error']) ?? -1) {
+            $errorCode = (int) ($content['error'] ?? $content['']['error'] ?? $content[$phone]['error']) ?? -1;
```

Comment:
this is duplicated.

```diff
-        if (0 !== (int) ($content['error'] ?? $content['']['error'] ?? $content[$phone]['error']) ?? -1) {
-            $errorCode = (int) ($content['error'] ?? $content['']['error'] ?? $content[$phone]['error']) ?? -1;
+        $errorCode = (int) ($content['error'] ?? $content['']['error'] ?? $content[$phone]['error']) ?? -1;
+        if (0 !== $code) {
```

Reply:
In line two see mistake in word $code, but found and fixed :+1: